zebra-chain/src/sprout/keys.rs

```diff
@@ -37,17 +38,17 @@ mod sk_magics {
 /// https://zips.z.cash/protocol/protocol.pdf#sproutkeycomponents
 fn prf_addr(x: [u8; 32], t: u8) -> [u8; 32] {
     let mut state = [0u32; 8];
-    let mut block = [0u8; 64];
+    let mut block: GenericArray<u8, U64> = GenericArray::default();
```

Comment:
I kept the type signature explict but rust compiler is able to infer it. I will remove it if it's unnecessary.

Reply:
We normally prefer leaning on type inference where possible but its fine to type it out explicitly if you prefer.

Alternatively, we can keep the types explicit while deduplicating the identifiers.

```suggestion
    let mut block = GenericArray::<u8, U64>::default();
```